sova/dashboard/routers/dependabot.py

```
@@ -0,0 +1,107 @@
+
+    try:
+        results = await sweep_dependabot_prs(
+            project_dir=get_project_dir(),
+            repo=cfg.github_repo,
+            github_user=cfg.github_user,
+            config=cfg.dependabot,
+            notification_config=cfg.notification,
+        )
+    except Exception:
+        log.warning("dependabot.sweep.error", exc_info=True)
+        raise HTTPException(status_code=500, detail="Sweep failed")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)